doc/example_code/how_to_examples/joystick_controller.rst

```diff
@@ -0,0 +1,169 @@
+.. _example-code:
+
+How To Get Connected Controllers
```

Comment:
This section should be using `arcade.get_controllers()` instead of `arcade.get_joysticks()`. The joysticks function is likely to be removed in Arcade 3.0.

It would also be good to see some coverage of `arcade.ControllerManager` here, which can be helpful for dynamically handling connect/disconnect events.